docs/release-notes/8.10.asciidoc

```diff
@@ -86,7 +86,7 @@ There are no user-facing changes in 8.10.2.
 [discrete]
 [[breaking-changes-8.10.0]]
 ==== Breaking changes
-There are no breaking changes in 8.10.0.
+* Deploying Elastic Defend as a DaemonSet in Kubernetes is no longer supported.
```

Comment:
Just noticed this -- so was this removal of support backported to 8.10? Typically I'd expect the previous versions to support something, and then in the current version we stop supporting it.

But if we're removing support even from previous versions, then we'd need to update ALL the release notes files for all the versions that we no longer support (so make the same change to 8.11.asciidoc, 8.12.asciidoc, etc.).

Reply:
The removal of support started with 8.10, but wasn't documented at the time. That's why I think it makes sense to just add it to the 8.10 RNs. Nothing about this feature changed in the 8.11 etc. versions

Reply:
Gotcha, makes sense. Thanks!